docs/Explore Algorithms/Causal Inference/Quickstart - Synthetic difference in differences.ipynb

```diff
@@ -0,0 +1,7533 @@
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "outputs": [],
+      "source": [
+        "unit_weights = model3.unitWeights.toPandas().set_index(\"state\")\n",
+        "unit_intercept = model3.unitIntercept\n",
+        "\n",
+        "time_weights = model3.timeWeights.toPandas().set_index(\"year\")\n",
+        "time_intercept = model3.timeIntercept\n",
+        "\n",
+        "pdf = df.toPandas()\n",
+        "pivot_df_control = pdf.query(\"~california\").pivot(\n",
+        "    index=\"year\", columns=\"state\", values=\"cigsale\"\n",
+        ")\n",
+        "pivot_df_treat = pdf.query(\"california\").pivot(\n",
+        "    index=\"year\", columns=\"state\", values=\"cigsale\"\n",
+        ")\n",
+        "sc_did = pivot_df_control.values @ unit_weights.values\n",
+        "treated_mean = pivot_df_treat.mean(axis=1)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "outputs": [],
+      "source": [
+        "fig, (ax1, ax2) = plt.subplots(\n",
+        "    2, 1, figsize=(15, 8), sharex=True, gridspec_kw={\"height_ratios\": [3, 1]}\n",
+        ")\n",
+        "fig.suptitle(\"Synthetic Diff in Diff Estimation\")\n",
+        "\n",
+        "ax1.plot(\n",
+        "    pivot_df_control.mean(axis=1), lw=3, color=\"C1\", ls=\"dashed\", label=\"Control Avg.\"\n",
+        ")\n",
+        "ax1.plot(treated_mean, lw=3, color=\"C0\", label=\"California\")\n",
+        "ax1.plot(\n",
+        "    pivot_df_control.index,\n",
+        "    sc_did,\n",
+        "    label=\"Synthetic Control (SDID)\",\n",
+        "    color=\"C1\",\n",
+        "    alpha=0.8,\n",
+        ")\n",
+        "ax1.set_ylabel(\"Cigarette Sales\")\n",
+        "ax1.vlines(\n",
+        "    1989,\n",
+        "    treated_mean.min(),\n",
+        "    treated_mean.max(),\n",
+        "    color=\"black\",\n",
+        "    ls=\"dotted\",\n",
+        "    label=\"Prop. 99\",\n",
+        ")\n",
+        "ax1.legend()\n",
+        "\n",
+        "ax2.bar(time_weights.index, time_weights[\"value\"], color=\"skyblue\")\n",
+        "ax2.set_ylabel(\"Time Weights\")\n",
+        "ax2.set_xlabel(\"Time\")\n",
+        "ax2.vlines(1989, 0, 1, color=\"black\", ls=\"dotted\")"
+      ]
+    }
+  ],
+  "metadata": {
+    "kernel_info": {
+      "name": "synapse_pyspark"
+    },
+    "kernelspec": {
+      "display_name": "Synapse PySpark",
+      "language": "Python",
+      "name": "synapse_pyspark"
+    },
+    "language_info": {
+      "name": "python"
+    },
+    "save_output": true,
+    "synapse_widget": {
```

Comment:
can you remove the output so it doesent blow up size of NB?

Reply:
Done.